tests/unit/components/search-autocomplete/SearchAutocomplete.spec.jsx

```diff
@@ -1,13 +1,24 @@
-import { vi } from 'vitest'
-import { render, fireEvent, screen, waitFor } from '@testing-library/react'
+import { describe, vi } from 'vitest'
+import {
+  render,
+  fireEvent,
+  screen,
+  waitFor as _waitFor
+} from '@testing-library/react'
 import SearchAutocomplete from '~/components/search-autocomplete/SearchAutocomplete'
 import userEvent from '@testing-library/user-event'
+import merge from 'lodash/merge'
 
-describe('SearchAutocomplete', () => {
-  const options = ['Finland', 'France', 'Georgia', 'Germany']
+const waitForTimeout = (callback, options) => {
+  const mergedOptions = merge({ timeout: 5000 }, options)
```

Comment:
can you use spread instead of merge from loadash?:

const mergedOptions = { timeout: 5000, ...options };

Also, maybe better to move this function to test-utils

Reply:
you are right, thanks.
Done!